src/main/kotlin/no/nav/syfo/application/ApplicationUtils.kt

```diff
@@ -9,7 +10,7 @@ private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.application")
 fun launchBackgroundTask(
     applicationState: ApplicationState,
     action: suspend CoroutineScope.() -> Unit,
-): Job = GlobalScope.launch {
+): Job = GlobalScope.launch(Dispatchers.Unbounded) {
```

Comment:
Forslag til kommentar: 
// Use Dispatchers.Unbounded to allow unlimited number of coroutines to be dispatched. Without this
// only a few will be allowed (depending on the number of availbale cores) which may result in cronjobs
// or Kafka-consumers not starting as intended.